src/main/frontend/format/block.cljs

```diff
@@ -341,7 +341,7 @@
           refs (distinct (concat (:refs block) ref-blocks))]
       (assoc block :refs refs))))
 
-(defn block-keywordize
```

Comment:
For my parsing spike, I needed most of this namespace so I had to fork it. Great to see this ns is getting improvements